src/Appwrite/Auth/Auth.php

```diff
@@ -110,6 +110,13 @@ class Auth
      */
     public static $secret = '';
 
+    /**
+     * User session ID.
+     *
+     * @var string
+     */
+    public static $sessionId = '';
+
```

Comment:
Oye..I didn't realize we don't have this yet...

@TorstenDittmann, is this okay so that we:

1. prevent finding the session using the secret multiple times as it can be expensive to iterate over every session and verify the secret hash
2. not rely on a non-existent session secret when a JWT is provided.